### PR TITLE
Add support for individually setting an item as deletable

### PR DESCRIPTION
**Pull request by go-diego-go**
This adds support for a function that checks if an item should be made deletable or not, independent of the rest
This new property is optional, and if missing the `deletable` property will be used instead (as is already the case)
- [x] @jfer-rubio

### CODE REVIEW

**Comment by jfer-rubio** [comment]
ufff 👌

File: dist/react-select.js [hunk 1]
```diff
@@ -1128,7 +1129,8 @@ var Value = React.createClass({
 		optionLabelClick: React.PropTypes.bool, // indicates if onOptionLabelClick should be handled
 		renderer: React.PropTypes.func, // method to render option label passed to ReactSelect
 		deletable: React.PropTypes.bool, // indicates if the value can be deleted
-		overlay: React.PropTypes.node // Popover overlay for deletable X icon
+		overlay: React.PropTypes.node, // Popover overlay for deletable X icon
+		isItemDeletable: React.PropTypes.funct // method to individually check if an item is deletable
```
Comment (codinronan) [inline dist/react-select.js]: Should be `React.PropTypes.func` (typo)

File: src/Value.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ var Value = React.createClass({
 		optionLabelClick: React.PropTypes.bool,           // indicates if onOptionLabelClick should be handled
 		renderer: React.PropTypes.func,                   // method to render option label passed to ReactSelect
 		deletable: React.PropTypes.bool,					  // indicates if the value can be deleted
-		overlay: React.PropTypes.node               // Popover overlay for deletable X icon
+		overlay: React.PropTypes.node,               // Popover overlay for deletable X icon
+		isItemDeletable: React.PropTypes.funct // method to individually check if an item is deletable
```
Comment (codinronan) [inline src/Value.js]: Oh, I guess it actually needs to be fixed here.

Comment (go-diego-go) [inline src/Value.js]: oh right that's wrong, thanks for catching it!

Comment (codinronan) [inline src/Value.js]: happy to help :)

**Comment by go-diego-go** [comment]
@ronanwize done

**Comment by codinronan** [comment]
LGTM!

**Pull request merged**